src/sleep.js

```diff
@@ -7,19 +7,19 @@ class Sleep {
     let allSleepData = this.sleepInfo.reduce((acc, user) => {
       acc += user[stat] / this.sleepInfo.length
       return acc 
-    }, 0)
-    return Number(allSleepData.toFixed(1))
+    }, 0);
+    return Number(allSleepData.toFixed(1));
   }
 
   getMetricByDate(date, metric) {
     let findDateAndMetric = this.sleepInfo.find(user => user.date === date);
-    return findDateAndMetric[metric]
+    return findDateAndMetric[metric];
   }
 
   getMetricByWeek(date, metric) {
-    let findDateIndex = this.sleepInfo.find(user => user.date === date);
-    let findMetric = this.sleepInfo.splice(findDateIndex - 6).map(user => user[metric]);
-    return findMetric
+    let findDateIndex = this.sleepInfo.findIndex(user => user.date === date);
+    let findMetric = this.sleepInfo.splice(findDateIndex - 6, 7).map(user => user[metric]);
+    return findMetric;
```

Comment:
@Garrett-Iannuzzi Thank you for working through to refactor this one with me once we realized it wasn't doing what we thought it was doing.   I'd like to see us refactor and change splice to the slice method so its accomplishing the same goal without mutating our data set.  We can look at this together tomorrow.